_examples/ThisClassMember.md

```diff
@@ -0,0 +1,68 @@
+---
+title: this class member
+---
+    
+**this** keyword, inside a class, refers to the instance. By default, used without identifiers, is referred to the current, innermost scope, the class where is declared. 
```

Comment:
I think there are a few too many commas. 

By default when used without identifiers, **this** refers to the current innermost scope where the class is declared.